test/composableStable.integration.test.ts

```diff
@@ -0,0 +1,107 @@
+// pnpm test -- test/composableStable.integration.test.ts
+import { beforeEach, describe, expect, test } from "vitest";
+import dotenv from "dotenv";
+dotenv.config();
+
+import { SmartOrderRouter } from "../src/sor";
+import { sorGetSwapsWithPools } from "../src/static";
+import { ChainId, BATCHSIZE, VAULT } from "../src/utils";
+import { Token, TokenAmount } from "../src/entities";
+import { OnChainPoolDataEnricher } from "../src/data/enrichers/onChainPoolDataEnricher";
+import { SwapKind, SwapOptions } from "../src/types";
+import { BasePool } from "../src/entities/pools";
+import { MockPoolProvider } from "./lib/utils/mockPoolProvider";
+
+import testPools from "./lib/testData/testPools/composableStable_17473810.json";
+import { RawStablePool } from "../src";
+
+describe("ComposableStable Swap tests", () => {
+  const chainId = ChainId.MAINNET;
+  const rpcUrl = "http://127.0.0.1:8545";
```

Comment:
If you refactor NETWORKS to an object, here you can replace rpcUrl by: 
`NETWORKS.MAINNET.rpcUrl` (I would rename NETWORKS to ANVIL_NETWORKS or alike)